src/Client/SidebarComponents/Navbar.fs

```diff
@@ -36,6 +36,17 @@ let private shortCutIconList model dispatch =
                 SpreadsheetInterface.CreateAnnotationTable ctrl |> InterfaceMsg |> dispatch
             )
         )
+        QuickAccessButton.create(
```

Comment:
Converting between term and unit is a relatively rare usecase. I think it is better to keep the quicka ccess buttons for common use cases.

Reply:
fixed